### PR TITLE
Bug/migrations table name

### PR DESCRIPTION
**Pull request by rodrigowirth**

### CODE REVIEW

File: spec/connect-middleware.spec.js [hunk 1]
```diff
@@ -56,5 +56,29 @@ describe('connect-middleware with default settings', function() {
...
     });
+
+    it('should create the migrations table for tenant 01', function (done) {
+      knex('01_knex_migrations')
+        .select()
+        .then(function (result) {
+          expect(result).to.have.length.above(1);
+          done();
+        })
+        .catch(function (err) {
```
Comment (maxclaus) [inline spec/connect-middleware.spec.js]: Could pass the done as argument. Instead of calling it manually.

File: spec/connect-middleware.spec.js [hunk 2]
```diff
@@ -56,5 +56,29 @@ describe('connect-middleware with default settings', function() {
...
+    });
+
+    it('should create the migrations table for tenant 02', function (done) {
+      knex('02_knex_migrations')
+        .select()
+        .then(function (result) {
+          expect(result).to.have.length.above(1);
+          done();
+        })
+        .catch(function (err) {
```
Comment (maxclaus) [inline spec/connect-middleware.spec.js]: Could pass the done as argument. Instead of calling it manually.

**Review by maxclaus — CHANGES_REQUESTED** [review]
Just a some few changes to improve testing code. But the code looks good. Can go for it once those changes are addressed.

File: spec/connect-middleware.spec.js (hunk 1 above)
Comment (rodrigowirth) [reply spec/connect-middleware.spec.js]: :+1:

File: spec/connect-middleware.spec.js (hunk 2 above)
Comment (rodrigowirth) [reply spec/connect-middleware.spec.js]: :+1:

**Pull request merged**